src/test/ui/parser/float-field-interpolated.stderr

```diff
@@ -13,7 +13,10 @@ error: expected one of `.`, `;`, `?`, `}`, or an operator, found `1.1`
   --> $DIR/float-field-interpolated.rs:8:13
    |
 LL |         { s.$b; }
```

Comment:
We probably don't want to suggest removing tokens if they're macro expansion variables.

Reply:
How would i check if it's an expansion variable? Maybe via the $ prefix?

Reply:
Okay, done!